packages/fscan/PKGBUILD

```diff
@@ -0,0 +1,41 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=fscan
+pkgver=117.2e452a9
+pkgrel=1
+pkgdesc='A Security Auditing Tool.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-scanner')
+url='https://github.com/shadow1ng/fscan'
+license=('custom:unknown')
+makedepends=('git' 'go')
+source=("git+https://github.com/shadow1ng/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
```

Comment:
Please use tags when available, as mentioned in our templates https://github.com/BlackArch/blackarch-pkgbuilds/blob/057a20ced2a87770fb83d169c56956fab9055ccb/PKGBUILD-generic-git#L17-L25